device_manufacturer_partnerships/explores/shipment_and_activation_vivo.explore.lkml

```diff
@@ -0,0 +1,6 @@
+include: "../views/shipment_and_activation_vivo.view.lkml"
+
+explore: shipment_and_activation_vivo {
+  label: "Vivo Distribution Shipments and Activations"
+  from: shipment_and_activation_vivo
```

Comment:
If your explore has the same name as the view then you can omit the `from` here.